src/Tribe/Editor/Template/Overwrite.php

```diff
@@ -129,14 +134,13 @@ public function has_early_access_to_blocks() {
 	 * @return bool
 	 */
 	public function should_inject_tickets_in_other_types( $post_id ) {
-		// Make sure this executed inside of the loop to prevent multiple requests and false positives of IDs
-		$in_the_loop = isset( $GLOBALS['wp_query']->in_the_loop ) && $GLOBALS['wp_query']->in_the_loop;
-
-		if ( empty( $post_id ) || is_admin() || ! $in_the_loop ) {
+		if ( empty( $post_id ) || is_admin()  ) {
```

Comment:
Notes: `in_the_loop` was false for posts, pages, and event in all of the iterations I tested ❓